app/src/components/buttons/multiActionButton/multiActionButton.scss

```diff
@@ -0,0 +1,148 @@
+.multi-action-button {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 30px;
+  box-sizing: border-box;
+  cursor: pointer;
+
+  &.disabled {
+    opacity: 0.4;
+    cursor: default;
+    pointer-events: none;
+  }
+  &.opened {
+    .toggle-icon {
+      transform: rotate(180deg);
+    }
+    .menu {
+      display: block;
+    }
+  }
+  &.color-booger {
+    .button-part {
+      background-color: $COLOR--booger;
+
+      &:hover {
+        background-color: darken($COLOR--booger, 15%);
+      }
+      &:active {
+        background-color: darken($COLOR--booger, 30%);
+      }
+    }
+
+    .menu-part {
+      background-color: $COLOR--booger;
+
+      &:hover {
+        background-color: darken($COLOR--booger, 15%);
+      }
+    }
+  }
+
+  &.color-organish {
+    .button-part {
+      background-color: $COLOR--organish;
+
+      &:hover {
+        background-color: darken($COLOR--organish, 15%);
+      }
+      &:active {
+        background-color: darken($COLOR--organish, 30%);
+      }
+    }
+
+    .menu-part {
+      background-color: $COLOR--organish;
+
+      &:hover {
+        background-color: darken($COLOR--organish, 15%);
+      }
+    }
+  }
+}
+
+.button-part {
+  display: block;
+  height: 100%;
+  padding: 0 10px;
+  border: none;
+  box-sizing: border-box;
```

Comment:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/
Please check all styles

Reply:
Fixed